examples/multimodal/components/frontend.py

```diff
@@ -40,6 +41,18 @@ class Frontend:
 
     @api(name="v1/chat/completions")
     async def generate(self, request: MultiModalRequest):
+        class_name = self.__class__.__name__
+        self.engine_args = parse_vllm_args(class_name, "")
+        self.model = self.engine_args.model
+
+        if self.model != request.model:
+            return StreamingResponse(
+                "Model not found",
+                media_type="text/event-stream",
+                status_code=400,
```

Comment:
I am conflicted here. The 400 HTTP code means "Bad Request", which it is. However, the 404 HTTP code means "Not Found" and the message returned is "Model not found".

Seems like `status_code=404` is more accurate. Is there a reason for not using it?

Reply:
Thanks for the catch, should be 404 instead of 400. Updated!